### DISCUSSION TITLE
ss-tproxy status 可能的坑

### DISCUSSION THREAD

**Discussion by GF-Huang** [zfl9/ss-tproxy#222] [Q&A]
非 `root` 权限下查询 `status` 的结果会让人误解。

![image](https://user-images.githubusercontent.com/4510984/103178703-1c42e980-48c0-11eb-8b5c-2ce493743a5d.png)

**Comment by zfl9** [comment]
这个忘了在文档说明，ss-tproxy绝大多数指令都得root权限。。
